addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -24,13 +32,25 @@ const GalleryWidget = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     _onClickImg: function (ev) {
-        if (this.$modal || ev.currentTarget.matches("a > img")) {
+        const clickedEl = ev.currentTarget;
+        if (this.$modal || clickedEl.matches("a > img")) {
             return;
         }
         var self = this;
-        var $cur = $(ev.currentTarget);
 
-        var $images = $cur.closest('.s_image_gallery').find('img');
+        let imageEls = this.target.querySelectorAll("img");
```

Comment:
I think target is there for compatibility reason. Use `this.el` instead. Sorry, I misled you by talking about target.